test/Jamfile

```diff
@@ -53,7 +59,7 @@ rule test_all
 
     if ! [ os.environ BOOST_SCOPE_TEST_WITHOUT_SELF_CONTAINED_HEADER_TESTS ]
     {
-        local headers_path = [ path.make $(BOOST_ROOT)/libs/scope/include/boost/scope ] ;
+        local headers_path = [ path.make $(SCOPE_INCLUDE_DIR)/boost/scope ] ;
```

Comment:
This could be changed just to "../boost/scope", right? `SCOPE_INCLUDE_DIR` seems to be unused anywhere else.